x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable_factory.ts

```diff
@@ -96,11 +97,14 @@ export class EmbeddableFactory implements EmbeddableFactoryDefinition {
         indexPatternService,
         timefilter,
         expressionRenderer,
-        editable: await this.isEditable(),
         basePath: coreHttp.basePath,
         getTrigger: uiActions?.getTrigger,
         getTriggerCompatibleActions: uiActions?.getTriggerCompatibleActions,
         documentToExpression,
+        capabilities: {
+          dashboardSave: Boolean(capabilities.dashboard?.showWriteControls),
```

Comment:
nit: might be nice to name `dashboardSave` and `visualizeSave` to a name that implies that their boolean type like `canSaveDashboard` or something like that

Reply:
Good call, I have renamed this!